include/openPMD/RecordComponent.hpp

```diff
@@ -69,6 +69,8 @@ namespace internal
          * Chunk reading/writing requests on the contained dataset.
          */
         std::queue<IOTask> m_chunks;
+
+        auto push_chunk(IOTask &&task) -> void;
```

Comment:
Simple?
```suggestion
        void push_chunk(IOTask &&task);
```

Reply:
I've switched to using mostly the trailing style, even for return types as simple as this, because functions align much more nicely that way and declarations are easier to visually parse at a first glance, see [here](https://github.com/openPMD/openPMD-api/blob/19395b09479f75eb992c01bcaac0f1165a371ea3/include/openPMD/snapshots/ContainerImpls.hpp) for example.
Since the rest of this class does not follow that style, I'll use your suggestion anyway.